.mergify.yml

```diff
@@ -1,13 +1,43 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
+
 pull_request_rules:
-- name: automatically merge scala-steward's PRs
-  actions:
-    merge:
-      method: squash
+- name: merge scala-steward's PRs
   conditions:
   - author=scala-steward
-  - body~=labels:.*semver-patch.*
+  - body~=labels:.*early-semver-patch
   - status-success=Build and Test (ubuntu-latest, 2.12.15, temurin@11)
   - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@11)
   - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@17)
   - status-success=Build and Test (ubuntu-latest, 3.1.1, temurin@11)
-  - status-success=Generate Site (ubuntu-latest, 2.13.8, temurin@11)
```

Comment:
The plugin automatically keeps these up-to-date when the Scala version or CI matrix changes.

Reply:
Woops, it shouldn't omit the site job. Easy fix.